### PR TITLE
Medal voting

### PR DESCRIPTION
**Pull request by Tanza3D**

### CODE REVIEW

File: medals/js/functions.js [hunk 1]
```diff
@@ -1268,3 +1277,27 @@ function randomMedal() {
     run(4, 0);
 }
 
+
+document.getElementById("vote-range-text").innerHTML = document.getElementById("vote-range").value;
```
Comment (EXtremeExploit) [inline medals/js/functions.js:1281]: use textContent

File: medals/js/functions.js [hunk 2]
```diff
@@ -1268,3 +1277,27 @@ function randomMedal() {
     run(4, 0);
 }
 
+
+document.getElementById("vote-range-text").innerHTML = document.getElementById("vote-range").value;
+
+document.getElementById("vote-range").addEventListener("input", () => {
+    document.getElementById("vote-range-text").innerHTML = document.getElementById("vote-range").value;
```
Comment (EXtremeExploit) [inline medals/js/functions.js:1284]: same here

File: medals/js/functions.js [hunk 3]
```diff
@@ -1268,3 +1277,27 @@ function randomMedal() {
...
+    data.append("value", document.getElementById("vote-range").value);
+
+    xhr.open("POST", "/medals/api/vote_medal.php", true)
+    xhr.send(data);
+    colMedals[medalname].HasVoted = true;
+    
+    document.getElementById("votepanel").classList.add("hidden");
+    
+    document.getElementById("votepanel-thanks").classList.remove("hidden");
+})
```
Comment (EXtremeExploit) [inline medals/js/functions.js:1303]: add new line and ;

File: medals/js/functions.js (hunk 1 above)
Comment (Tanza3D) [reply medals/js/functions.js:1281]: can i use innerText i use that everywhere else

Comment (EXtremeExploit) [reply medals/js/functions.js:1281]: sure

**Pull request merged**